src/gui-wizard-gtk/wizard.c

```diff
@@ -2155,13 +2155,13 @@ static void start_event_run(const char *event_name)
 
     if (prepare_commands(state, g_dump_dir_name, event_name) == 0)
     {
- no_cmds:
-        /* No commands needed?! (This is untypical) */
-        free_run_event_state(state);
-        char *msg = xasprintf(_("No processing for event '%s' is defined"), event_name);
+        /* No commands found for this event. Acknowledge this fact and continue
+         * with processing. */
+        /* TODO: Perhaps don't even show this to the user, only log via VERB1/2 log. */
```

Comment:
I don’t know, “reacting to this situation in the GUI” will be pretty much the same, no? Although maybe without the event name.